shared/C.js

```diff
@@ -3756,7 +3756,8 @@ Array.prototype.pushAll = function(otherArray) {
 			{ match : {name : "Howlpack Alpha"}, replace : {layout : "double-faced", names : ["Mayor of Avabruck", "Howlpack Alpha"]}},
 			{ match : {name : "Overbeing of Myth"}, replace : {flavor : "\"She walks among us unseen, learning from our imperfections.\"\n—The Seer's Parables"}},
 			{ match : {name : "Wren's Run Packmaster"}, replace : {releaseDate : "2007-09-29"}},
-			{ match : {name : "Sultai Ascendacy"}, replace : {name : "Sultai Ascendancy", imageName : "sultai ascendancy"}}
+			{ match : {name : "Sultai Ascendacy"}, replace : {name : "Sultai Ascendancy", imageName : "sultai ascendancy"}},
+			{ match : {name : "Nissa, Vastwood Seer"}, replace : {artist : "Wesley Burt"}}
```

Comment:
MagicCards.info doesn't have the artist listed and results in the card having an empty string for artist. This fixes that.